### PR TITLE
VACMS-11421: add tricare description to service taxonomy

### LINKED ISSUE #11421: CMS: Add tricare description field to service taxonomy.
## Description
As a TRICARE person, I want there to be centralized service descriptions specific to TRICARE in the services taxonomy so that the FE can use them and they can be managed by the content team separately from the facility editor.

## Acceptance Criteria
- [x] field_tricare_description exists as a field on health_care_service_taxonomy
   - [x] label : TRICARE service description
   - [x] type: text plain long
   - [x] form location 
image
   - [x] Widget Text area multiple rows
   - [x] cardinality 1
   - [x] not required
   - [x]  display location 
image
- [ ] Behat spec tests updated

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by JayDarnell**
## Description

Closes #11421 

## Testing done

Manual testing

## Screenshots

Screen Shot 2022-11-03 at 2 12 26 PM

Screen Shot 2022-11-03 at 2 12 54 PM

Screen Shot 2022-11-03 at 2 11 09 PM

## QA steps

As an admin user
- [x] Visit /admin/structure/taxonomy/manage/health_care_service_taxonomy/overview/fields
- [x] field_tricare_description exists as a field on health_care_service_taxonomy
   - [x] label : TRICARE service description
   - [x] type: text plain long
   - [x] form location 
image
   - [x] Widget Text area multiple rows
   - [x] cardinality 1
   - [x] not required
   - [x]  display location 
image

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

### CODE REVIEW

File: config/sync/field.field.taxonomy_term.health_care_service_taxonomy.field_tricare_description.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - epp
+third_party_settings:
+  epp:
+    value: ''
+    on_update: 0
+id: taxonomy_term.health_care_service_taxonomy.field_tricare_description
+field_name: field_tricare_description
+entity_type: taxonomy_term
+bundle: health_care_service_taxonomy
+label: 'TRICARE Service Description'
```
Comment (swirtSJW) [inline config/sync/field.field.taxonomy_term.health_care_service_taxonomy.field_tricare_description.yml:18]:
```suggestion
label: 'TRICARE service description'
```

image


image

File: tests/behat/drupal-spec-tool/content_model_vocabulary_fields.feature [hunk 2]
```diff
@@ -27,6 +27,7 @@ Feature: Content model: Vocabulary fields
 | Vocabulary | VA Services | Show for VAMC Facilities | field_show_for_vamc_facilities | Boolean |  | 1 | Single on/off checkbox |  |
 | Vocabulary | VA Services | Show for VBA Facilities | field_show_for_vba_facilities | Boolean |  | 1 | Single on/off checkbox |  |
 | Vocabulary | VA Services | Show for Vet Centers | field_show_for_vet_centers | Boolean |  | 1 | Single on/off checkbox |  |
+| Vocabulary | VA Services | TRICARE Service Description | field_tricare_description | Text (plain, long) |  | 1 | Text area (multiple rows) |  |
```
Comment (swirtSJW) [inline tests/behat/drupal-spec-tool/content_model_vocabulary_fields.feature:30]:
```suggestion
| Vocabulary | VA Services | TRICARE service description | field_tricare_description | Text (plain, long) |  | 1 | Text area (multiple rows) |  |
```
You'll need to fix this in the spec tool.

**Review by swirtSJW — APPROVED** [review]
Minor but necessary.  VA is fanatical about sentence casing.

**Review by swirtSJW — CHANGES_REQUESTED** [review]
I meant this one instead

**Review by omahane — DISMISSED** [review]
@JayDarnell This appears correct.
I'm unclear, however, why all but the "Description" field renders tags.
Screen Shot 2022-11-03 at 4 14 10 PM

**Review by swirtSJW — APPROVED** [review]

**Pull request merged**